provider/datagen/src/crabbake.rs

```diff
@@ -141,17 +155,61 @@ impl DataExporter<CrabbakeMarker> for ConstExporter {
                 quote! { (#options, #ident) }
             });
 
+            let module_path = syn::parse_str::<syn::Path>(
+                &key.get_path().replace(&['=', '@'], "_").replace('/', "::"),
```

Comment:
nit: probably have a comment explaining why this exists

Reply:
also i'd probably replace `@` with `_v`?

Also is there a reason `/` goes to `::`? wouldn't that make it necessary to use nested modules on the crabbake side? I guess that's fine